standup_notes/__main__.py

```diff
@@ -83,14 +90,56 @@ def copy_note(day: date):
         print(note + ' doesn\'t exist yet.')
 
 
+def copy_prev(day: date):
+    previous_days_note = get_note_name(last_weekday(day))
+    note = get_note_name(day)
+    date_of_note = "Date: " + day.strftime("%m/%d/%Y") + " \n"
+    beginning_format = "__What I did yesterday__:\n"
+    end_format = "__What I'm doing today__:\n \n__Blockers__:\n-none\n"
+    flag = False
+    lines_to_append = []
+
+    if os.path.exists(previous_days_note):
+        result = verify_input("Found yesterdays notes, would you like to insert applicable information into your "
```

Comment:
As in it will take yesterdays "What I am doing today info" and not take "What I did yesterday" info

Reply:
Could you change the question a bit to describe that it'll take the previous day's "What I am doing today" section? I'm not sure "application information" is clear enough. Also, it may not actually be "yesterday". If you are trying to edit tomorrow's note and include the copy flag, then it'll actually be copying from today's note, not yesterday.